test/standalone/test_rpc_proxy.py

```diff
@@ -1,31 +1,30 @@
 import socket
+from contextlib import contextmanager
 
 import eventlet
 import pytest
 from eventlet.event import Event
 from kombu.connection import Connection
 from kombu.message import Message
-from mock import Mock, call
+from mock import Mock, patch
+from six.moves import queue
 
+from nameko.amqp.consume import Consumer
+from nameko.constants import HEARTBEAT_CONFIG_KEY
 from nameko.containers import WorkerContext
-from nameko.exceptions import RemoteError, RpcTimeout
+from nameko.exceptions import (
+    RemoteError, ReplyQueueExpiredWithPendingReplies, RpcTimeout
+)
 from nameko.extensions import DependencyProvider
-from nameko.rpc import MethodProxy, Responder, get_rpc_exchange, rpc
+from nameko.rpc import Responder, get_rpc_exchange, rpc
 from nameko.standalone.rpc import (
-    ClusterRpcProxy, ConsumeEvent, ServiceRpcProxy
+    ClusterRpcProxy, ReplyListener, RpcProxy, ServiceRpcProxy
 )
 from nameko.testing.waiting import wait_for_call
 
 from test import skip_if_no_toxiproxy
 
 
-# uses autospec on method; needs newer mock for py3
-try:
-    from unittest.mock import patch
-except ImportError:  # pragma: no cover
-    from mock import patch
```

Comment:
`unittest.mock.patch` and `mock.patch` have different APIs (only one returns the patched object) so have reverted to using just one